### ISSUE TITLE
Adding Functions For Encryption and Decryption of Image

### ISSUE THREAD

**Issue by kailashchoudhary11** [kailashchoudhary11/imgman#52]
Add Function to Encrypt and Image which should take path of the image as an argument.
Decrypt Function To Decrypt the image and it must also take the path of image!

**Comment by LucasRochaAbraao** [comment]
Hey, I'm looking for an issue to contribute to, what do you mean by encrypting and decrypting an image? When will this function be called? I'm skimming through the repo, is it supposed to be a member method of the `ProcessImage` class in views.py?

**Comment by kailashchoudhary11** [comment]
> Hey, I'm looking for an issue to contribute to, what do you mean by encrypting and decrypting an image? When will this function be called? I'm skimming through the repo, is it supposed to be a member method of the `ProcessImage` class in views.py?

Hey Irocha you can add functions in the file called Helpers.py which is present in sinimg. Also you can google about encryption and decryption. Let me know if you would like to work on this

**Comment by mitanshudodia** [comment]
Hey @kailashchoudhary11 there is already encrypt_image and decrypt_image functions in Helpers.py file, so what exactly you want to do ?

**Comment by kailashchoudhary11** [comment]
> Hey @kailashchoudhary11 there is already encrypt_image and decrypt_image functions in Helpers.py file, so what exactly you want to do ?

Yep there is one But, there are many other techniques of encryption and decryption So you could implement any other

**Comment by mitanshudodia** [comment]
> > Hey @kailashchoudhary11 there is already encrypt_image and decrypt_image functions in Helpers.py file, so what exactly you want to do ?
> 
> Yep there is one But, there are many other techniques of encryption and decryption So you could implement any other

Ok assign this to me i will try to use cryptography module for encryption and decryption.

**Comment by kailashchoudhary11** [comment]
@LucasRochaAbraao @mitanshudodia You'll can join discord server and we can discuss over there. Link is in the Readme File

**Comment by bhoopesh369** [comment]
Can i work on this issue?

**Comment by bhoopesh369** [comment]
@kailashchoudhary11

**Comment by kailashchoudhary11** [comment]
> @kailashchoudhary11

Yep

**Comment by kailashchoudhary11** [comment]
> @kailashchoudhary11

which algorithm will you be implementing?

**Comment by bhoopesh369** [comment]
Using pycrypto

**Comment by kailashchoudhary11** [comment]
Okay Go Ahead ! @bhoopesh369

**Comment by bhoopesh369** [comment]
PR #58

**Comment by bhoopesh369** [comment]
@kailashchoudhary11

**Comment by bhoopesh369** [comment]
PR #60
